cfgmgr/vrfmgr.cpp

```diff
@@ -63,6 +64,18 @@ VrfMgr::VrfMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, con
                 break;
         }
     }
+
+    cmd.str("");
+    cmd.clear();
+    cmd << IP_CMD << " rule | grep '^0:'";
+    if (swss::exec(cmd.str(), res) == 0)
+    {
+        cmd.str("");
+        cmd.clear();
+        cmd << IP_CMD << " rule add pref 1001 table local && " << IP_CMD << " rule del pref 0 && "
```

Comment:
This might break in case forced_mgmt_routes is configured. 
We need corresponding changes in interfaces.j2 to ensure rule for forced_mgmt_routes is below table local, else default route lookup fails since default route will be installed in local table.

Reply:
@preetham-singh Why should the pref of the rule for mgmt_routes be below table local(1001)?  Could you describe it more detail? Per the existing interface.j2 file mgmt_routes with vrf attribute should be in table (5000) whose pref is 1000. As we know default route on specified vrf should in installed in own route table and global default route will be put in table default.

Reply:
Please ignore this comment as forced_mgmt_routes inserts specific routes to default table.